File: app/src/main/java/com/c2/arenafinder/data/response/ArenaFinderResponse.java
package com.c2.arenafinder.data.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ArenaFinderResponse {

    @Expose
    @SerializedName("status")
    public String status;
    @Expose
    @SerializedName("message")
    public String message;

    public ArenaFinderResponse(String status, String message) {
        this.status = status;
        this.message = message;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: app/src/main/java/com/c2/arenafinder/data/response/ArenaFinderResponse.kt
package com.c2.arenafinder.data.response

import com.c2.arenafinder.data.model.ArenaFinderModel
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Digunakan untuk menerima response dari server pada data aplikasi arenafinder
 *
 */
data class ArenaFinderResponse(
    @Expose
    @SerializedName("status")
    var status : String?,
    @Expose
    @SerializedName("message")
    var message : String?,
    @Expose
    @SerializedName("data")
    var data : ArenaFinderModel
)
